### ISSUE TITLE
Slack bot for short queries

### ISSUE THREAD

**Issue by kberzinch** [RoboJackets/apiary#236]
From **Make a Wish:**
>Nothing for the site, but a Slack bot to interact with for quick queries and reminders. For example, "@!apiary did ___ pays dues this semester?", "@!apiary has ___ collected their polo?" "@!apiary what shirt size is ___?". There's probably a ton of other ideas that could be explored. This is not something that needs to happen, just a nice to have, it may not even work or be practical.

**Comment by ryanstrat** [comment]
Can we put this up as a Jr. Design project or something? I think it would be really fun to work on, has an isolated scope, and would get use.

**Comment by Arthelon** [comment]
yeah, sounds like a separate application that would hook up to the apiary API

**Comment by Arthelon** [comment]
I've created a Slack bot that implements a few simple queries. I'm currently hosting it on [glitch](https://apiary-bot.glitch.me) and you can find the source code [here](https://github.com/Arthelon/robojackets-apiary-slack-bot).

**Comment by JoshOldenburg** [comment]
Needs support for a `/override` to give people access overrides
